docs/hardware/devices/lora/index.mdx

```diff
@@ -145,6 +145,12 @@ Early versions of some of these boards contained the wrong component in the LiPo
   - Micro USB
   - Antenna: SMA antenna connector
 
+## Variations of identifying marks
+  - FCC ID: 2ASYE-T3-V1-6-1
+    - "LILYGO 868/915 Modle: LORA32" sticker on the chip
```

Comment:
The LORA32 sticker is on different boards too ... it's not exclusive here. Better remove that line.

Reply:
Hmm, I was trying to capture all of the markings. Like, if it has each of those things, it's a 2.1-1.6. I guess that's not clear though... any suggestions?

Reply:
Or are you saying that that sticker is so common that there's no point including mention of it at all. No one is likely to find a board with the other markings that _doesn't_ have that sticker on the chip?

Reply:
yes... all lilygo boards with LORA functonality have that particular sticker. or a different one saying 'SX1262' - but that's it. Doesn't really help with ID.